features/step_definitions/kata.rb

```diff
@@ -227,3 +227,39 @@
     end
   end
 end
+
+Given /^a valid cluster type for kata tests exists$/ do
+  accepted_platforms = ['gcp', 'azure']
+  iaas_type = env.iaas[:type] rescue nil
+  raise "Kata installation only supports GCE platform #{iaas_type} is not a valid cluster type" unless accepted_platforms.include? iaas_type
```

Comment:
The above line show `accepted_platforms` as gcp/azure, while here show GCE, mismatch.